src/plugins/thirdroom/thirdroom.game.ts

```diff
@@ -168,6 +171,29 @@ export const ThirdRoomModule = defineModule<GameState, ThirdRoomModuleState>({
 
     // await waitForRemoteResource(ctx, environmentMapTexture.resourceId);
 
+    // create out of bounds floor check
+    const { collisionHandlers, physicsWorld } = getModule(ctx, PhysicsModule);
+    const rigidBody = physicsWorld.createRigidBody(RAPIER.RigidBodyDesc.newStatic());
+    const size = 10000;
+    const colliderDesc = RAPIER.ColliderDesc.cuboid(size, 50, size)
+      .setActiveEvents(RAPIER.ActiveEvents.CONTACT_EVENTS)
+      .setCollisionGroups(0xf000_000f)
+      .setSolverGroups(0xf000_000f);
+    physicsWorld.createCollider(colliderDesc, rigidBody.handle);
+
+    rigidBody.setTranslation(new RAPIER.Vector3(size / 2, -150, size / 2), true);
```

Comment:
At some point we should make this size configurable by the content creator, but this seems safe for now.